src/cgame/cg_ents.cpp

```diff
@@ -1381,3 +1381,59 @@ void CG_AddPacketEntities()
 		}
 	}
 }
+
+// see also G_Team in src/sgame/sg_ents.cpp
+team_t CG_Team(const centity_t *ent)
```

Comment:
Why don't you use this function in any of these places where the team needs to be determined?

Reply:
I've moved the CG_Team work into another PR as it can wait for the 0.53 "AnyTeam" milestone. If anyone wants to tag it, it's https://github.com/Unvanquished/Unvanquished/pull/1337 (leet!)